### PR TITLE
Shorter title and description

### LINKED ISSUE #199: en-US title in Fastlane too long
[!] Google Api Error: forbidden: The title for the app is too long for language en-US. - The title for the app is too long for language en-US.

So I'm reverting to the older version for now.

Ping @comradekingu

### PR DESCRIPTION
**Pull request by comradekingu**
Fixes #199

### CODE REVIEW

**Comment by TheLastProject** [comment]
Did you see #217 and the post linked there? Emojis in the app titles are getting punished in search ranking by Google.

**Comment by comradekingu** [comment]
@TheLastProject Ah, I thought "second half of 2021" wasn't so imminent.

**Comment by comradekingu** [comment]
Meh, "discount" is not allowed, so probably "discounts" is not good.

File: fastlane/metadata/android/en-US/short_description.txt [hunk 1]
```diff
@@ -1 +1 @@
-No plastic pollution. 💳 Store discounts, membership cards and barcodes in this EDC wallet.
+Bring your barcodes, memberships, loyalty programs, coupons and tickets.
```
Comment (comradekingu) [inline fastlane/metadata/android/en-US/short_description.txt:1]:
```suggestion
For your barcodes, memberships, loyalty programs, coupons and tickets.
```

**Comment by TheLastProject** [comment]
That "Discount" part is related to "stuff not about your app". So, you shouldn't go "the app is 50% discounted now". But "manage your store discounts" is a whole different thing.

**Comment by comradekingu** [comment]
@TheLastProject I am thinking it is going to be some dumb filtering at play. It isn't that bad now, and there can be translations that sort of fit either way. Made a mental note to mention the descriptions and title should be local. So no "coupons" for areas that don't have them, etc.

**Comment by comradekingu** [comment]
@TheLastProject I think this is good. Wondering whether emojis need to be removed from the main text too.

**Pull request merged**